### PR TITLE
Mixpanel integration

### PR DESCRIPTION
**Pull request by renesansz**

### CODE REVIEW

**Comment by aaustin** [comment]
So, unfortunately this is a bit off. Apologies if this was unclear. Here is the spec:
1. Please remove all references to Mixpanel. We're not going to add Mixpanel to our SDK.
2. Please add the following new method to the library:
`setRequestMetadataKeyAndValue(key, value)`

Make this call the following methods in the library
iOS: `[[Branch getInstance] setRequestMetadataKey:key value:value];`
Android: `Branch.getInstance().setRequestMetadata(key, value);`

Note that this has nothing to do with Mixpanel at all.

**Comment by renesansz** [comment]
I see, thanks, that was also I was kind of confused on what you meant. will submit a follow up fix for this.

**Comment by renesansz** [comment]
Updated PR @aaustin , I apologize I misunderstood how it works 😄

File: README.md [hunk 1]
```diff
@@ -167,6 +167,7 @@ Branch.getFirstReferringParams().then(function (res) {
 1. Branch Session
   + [setDebug](#setDebug)
   + [initSession](#initSession)
+  + [getMixpanelInstance](#getMixpanelInstance)
```
Comment (aaustin) [inline README.md]: Can you change this method name to `setMixpanelToken` and update the references below?

Comment (renesansz) [inline README.md]: @aaustin , updated the api reference.

**Comment by aaustin** [comment]
👍

**Pull request merged**